tests/Keboola/SSHTunnel/SSHTest.php

```diff
@@ -38,6 +40,25 @@ public function testOpenTunnel()
         $this->assertEquals(0, $process->getExitCode());
     }
 
+    public function testOpenTunnelWithCompression()
+    {
+        $ssh = new SSH();
+
+        $process = $ssh->openTunnel([
+            'user' => 'root',
+            'sshHost' => 'sshproxy',
+            'sshPort' => '22',
+            'localPort' => '33306',
+            'remoteHost' => 'mysql',
+            'remotePort' => '3306',
+            'privateKey' => $this->getPrivateKey(),
+            'compression' => true
+        ]);
+
+        $this->assertContains('-fNC', $process->getCommandLine());
+        $this->assertEquals(0, $process->getExitCode());
```

Comment:
podle mého by se ten test měl zkusit připojit. Takhle je to více méně k ničemu. Resp. zkusil jsem si lokálně dát třeba port 3307 nebo `'remoteHost' => 'mysql2',` a projde to úplně stejně.

Reply:
No uplne k nicemu to neni, otestuje to, ze sa ten tunnel uspesne vytvori, to ze smeruje do prdele je druha vec :D
Ten test pridam.

Reply:
Jo, vlastně - sorry - "remoteHost" je cíl toho tunelu - ne ten SSH host! Já jsem to bral tak, že to vrátí nulu i když server ke kterému se to má připojit neexistuje...  

Ale každopádně - tohle neotestuje, že se správně vytvořil ten tunel. Když by tam byl třeba bug a udělal se ten tunel localhost:3307 -> mysql:3307 (typo - local port 3307 místo remote port 3306), tak to z toho testu nepoznáme.